packages/perseus/src/widgets/categorizer/categorizer.util.ts

```diff
@@ -0,0 +1,25 @@
+import type {PerseusCategorizerWidgetOptions} from "../../perseus-types";
+
+type CategorizerPublicWidgetOptions = {
+    // a list of items to categorize. e.g. ["banana", "yellow", "apple", "purple", "shirt"]
+    items: PerseusCategorizerWidgetOptions["items"];
+    // a list of categories. e.g. ["fruits", "colors", "clothing"]
+    categories: PerseusCategorizerWidgetOptions["categories"];
+    // Whether the items should be randomized
+    randomizeItems: PerseusCategorizerWidgetOptions["randomizeItems"];
+    // Whether this widget is displayed with the results and immutable
+    static: PerseusCategorizerWidgetOptions["static"];
+};
```

Comment:
I wasn't 100% sure on what should be kept here. I took out `values` since that is where the answer is. I also took out `highlightLint` and `linterContext` as those seemed related to the editors (and editors will get the full widget options). Let me know if those should go back though.

Reply:
I think you are on the right track. We want to remove everything that's in the associated `ScoringData` type if we can. I think we've found some widgets where the widget depends on some bits of scoring data to function and so we'll have to untangle those as we get to them. 

Removing `highlightLint` and `linterContext` are a good move. In fact, I suspect that those are not actually something we should be saving as part of the data, but rather props that are used in the editor at runtime (not something you need to fix now).